src/sentry/integrations/discord/webhooks/command.py

```diff
@@ -37,25 +41,35 @@ class DiscordCommandHandler(DiscordInteractionHandler):
 
     def handle(self) -> Response:
         command_name = self.request.get_command_name()
-        logging_data = self.request.logging_data
+        cmd_input = CommandInput(command_name)
+        dispatcher = MsTeamsCommandDispatcher(self.request)
```

Comment:
I'm a bit confused as why the class and file are discord but dispatcher is MsTeams 🤔

Reply:
Discussed in team meeting. It's a typo, so please update.